payjoin/src/psbt/mod.rs

```diff
@@ -1,5 +1,8 @@
 //! Utilities to make work with PSBTs easier
 
+/// Utilities for merging unique v0 PSBTs
+pub(crate) mod merge;
```

Comment:
This is where a `#[cfg(feature = "merge")]` gate would belong in my understanding of it

I think best practice is to replace the `///` docstring here with a `//!` module level documentation, of which the one line here is fine, in `merge.rs`

Reply:
Rats I should have suggested feature gating on `multiparty`. `merge` basically makes sense only in this interim, but not a big deal. we'll replace